source/reference/minio-mc/mc-idp-ldap-accesskey-disable.rst

```diff
@@ -0,0 +1,102 @@
+.. _minio-mc-idp-ldap-accesskey-disable:
+
+=================================
+``mc idp ldap accesskey disable``
+=================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+.. mc:: mc idp ldap accesskey disable
+
+
+Description
+-----------
+
+.. start-mc-idp-ldap-accesskey-disable-desc
+
+:mc:`mc idp ldap accesskey disable` disables the specified access key on the local server.
+
+.. end-mc-idp-ldap-accesskey-disable-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+         The following example disables the access key ``mykey`` from the ``minio`` deployment:
```

Comment:
```suggestion
         The following example disables the access key ``mykey`` on the ``minio`` deployment:
```

Reply:
I thought I'd caught all of those. Thanks.